examples/topics/categorical/ridgeplot.py

```diff
@@ -1,13 +1,21 @@
 ''' A `ridgeline plot`_ using the `Perceptions of Probability`_ dataset.
-This example demonstrates the uses of categorical offsets to position
-categorical values explicitly. This chart shows the distribution of responses
-to the prompt *What probability would you assign to the phrase "Highly likely"*.
+
+This example demonstrates the uses of categorical offsets to position categorical
+values explicitly, which in this case allows for "poor-man's" sub-plots. This is
```

Comment:
Avoiding idioms probably better for more people
```suggestion
values explicitly, which in this case allows for makeshift sub-plots. This is
```